react/features/stream-effects/rnnoise/RnnoiseProcessor.ts

```diff
@@ -66,22 +61,13 @@ export default class RnnoiseProcessor {
             // For VAD score purposes only allocate the buffers once and reuse them
             this._wasmPcmInput = this._wasmInterface._malloc(RNNOISE_BUFFER_SIZE);
 
+            // eslint-disable-next-line no-bitwise
```

Comment:
Feel free to disable it for the whole file.

Reply:
done